Batch8CucumberSeleniumAPI/pom.xml

```diff
@@ -108,12 +108,12 @@
 		<dependency>
 			<groupId>org.apache.logging.log4j</groupId>
 			<artifactId>log4j-api</artifactId>
-			<version>2.13.1</version>
```

Comment:
Why unnecessary you are making changes in dependencies

Reply:
i did the changes in dependency because it was giving error of " log4j version"

Reply:
ok so sometimes, if you want to change locally to resolve something , you can, but you do not have to commit it. Since pom file version for log4j is a local, u should have submitted only the change which is relevant task.
But dnt worry, thats how you learn, so now revert the changes in your pom file again, make it as before and then commit. Cheers!

Reply:
okay sir